css/styles.css

```diff
@@ -1,13 +1,126 @@
 *, *::before, *::after {
-  box-sizing: border-box;
+    box-sizing: border-box;
 }
 
 body {
     margin: 0;
     padding: 0;
+    background-image: radial-gradient(#2E4CF5, #4B91F2);
+    font-family: 'Special Elite', cursive;
 }
 
 img {
-  width: 100%;
-  height: auto;
-}
\ No newline at end of file
+    width: 100%;
+    height: auto;
+}
+
+header {
+    margin: 4rem 2rem 2rem;
+}
+
+h1 {
+    margin-top: 0.5rem;
+    text-align: center;
+
+}
+
+.collection {
+    width: 95vw;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
I don't think you need this, I believe grid has 1 column by default

Reply:
If I delete this it messes with my formatting from media queries later on - probably a more elegant way to do things but would involve more CSS shenanigans.